### PR TITLE
[15.0][MIG] sale_sourced_by_line

### PR DESCRIPTION
**Pull request by JoanMForgeFlow**
Standard migration 15.0
Red because: https://github.com/OCA/sale-workflow/pull/1937

### CODE REVIEW

**Review by celm1990 — APPROVED** [review]

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

File: sale_sourced_by_line/README.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@
 Sale Sourced by Line
 ====================
 
-This module adds the possibility to source a line of sale order from a specific
+This module adds the possibility to source a line meof sale order from a specific
```
Comment (mariadforgeflow) [inline sale_sourced_by_line/README.rst]: small typo here

**Comment by rousseldenis** [comment]
@JoanMForgeFlow Could you rebase ?

**Comment by AaronHForgeFlow** [comment]
Rebased

**Comment by JoanMForgeFlow** [comment]
@rousseldenis and @HaraldPanten or ping @pedrobaeza could you please merge this PR thanks in advance :)

File: sale_sourced_by_line/README.rst (hunk 1 above)
Comment (JoanMForgeFlow) [reply sale_sourced_by_line/README.rst]: Fixed, thanks for reporting ;)

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**